### ISSUE TITLE
Phenotype annotations to GO term

### ISSUE THREAD

**Issue by PCarme** [pombase/canto#2850]
While reviewing a community curation, I found several phenotype annotations to GO terms. 
<img width="624" alt="image" src="https://github.com/user-attachments/assets/395f1fd0-22f2-4c90-97a0-2f1ec0b440ec">
That seems weird, is it a bug ?

**Comment by ValWood** [comment]
very weird!

**Comment by kimrutherford** [comment]
It's possible (but not encouraged!) to paste an ontology term ID into the term name search box.  The ID is checked to make sure it's a real term ID but that's all.

I assume here that the community curator couldn't find the term they wanted for the WT function in FYPO, but did find it in GO?

**Comment by PCarme** [comment]
Yes, I realised after that it is possible to search for a GO ID in the search box and select it. 
They used it to annotate the term as a phenotype to a "wild type" allele, so my guess is they wanted to make a GO annotation and didn't find how, so they used the phenotype annotation instead.
